lib/hal/led.h

```diff
@@ -0,0 +1,15 @@
+#ifndef HAL_LED_H
+#define HAL_LED_H
+
+#include <xboxkrnl/xboxkrnl.h>
+#include <assert.h>
```

Comment:
`assert` is not used in the header. Only include what you use, and include it so it has the least side-effects possible.